knitr/reduce-sum/reduce_sum_tutorial.Rmd

```diff
@@ -286,12 +287,13 @@ Run and time the model with:
 ```{r}
 redcard_data$grainsize <- 1
 time1 = system.time(fit1 <- logistic1$sample(redcard_data,
-                                             num_cores = 4,
-                                             num_chains = 4,
+                                             cores = 4,
+                                             chains = 4,
                                              refresh = 1000))
 
 time1
 ```
+**Note:** Older versions of cmdstanr use `num_cores` and `num_chains` instead of `cores` and `chains`. If you get an error, [update](https://mc-stan.org/cmdstanr/) `cmdstanr`.
```

Comment:
There should just be a deprecation warning and not an error, but I guess eventually we will get rid of the deprecation warning and make it an error so fine by me to keep this Note.